### ISSUE TITLE
admin-ui: issues with having multiple values in an attribute that is managed via user profile

### ISSUE THREAD

**Issue by antikalk** [keycloak/keycloak#19578]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

user-profile

### Describe the bug

KC 21 brought some nice changes for the user details screen if user profile is enabled but I also discovered some issues.

1. Attributes are hidden from the Attributes Tab, only when the user profile query has been completed. When e.g. refreshing the page on the attributes tab you will still be able to view/edit user profile attributes here:
![image](https://user-images.githubusercontent.com/1793007/230431517-c65b55db-6060-451c-9f9d-ddcac7530212.png)
When refreshing on the details tab and then switching to the Attributes tab, the Attributes are not shown: 
![image](https://user-images.githubusercontent.com/1793007/230431699-55823da3-57ef-406e-8618-8e7632c0ea77.png)

2. When on the details screen the user attributes with multiple values are shown comma separated: 
![image](https://user-images.githubusercontent.com/1793007/230432075-98eb160e-d12a-4ac2-bfd1-9b87d0439065.png)
When editing and saving the value it is not send as different values anymore but as one value:
![image](https://user-images.githubusercontent.com/1793007/230432866-d9cf4447-32a5-4eec-adad-2a7c12d02eb4.png)
![image](https://user-images.githubusercontent.com/1793007/230432919-0a085b27-2f3c-4b18-9ff1-5825a5ca0a22.png)

### Version

21.0.2

### Expected behavior

1.  Consistent behavior of the attributes tab
2.  Being able to still edit multi value attributes with the new user profile UI. 

### Actual behavior

See description.

### How to Reproduce?

1. Activate user profile
2. Add user profile field for e.g. device_ids
3. Create a user and check the user details screen
4. Go to the attributes tab
5. Refresh the page and add multiple entries for `device_ids`

**Comment by edewit** [comment]
So you can only make it a multi value field if you have the bug with the fields showing up in the attributes tab, there is no UI to create user profile fields that are multi value, right?

**Comment by antikalk** [comment]
> So you can only make it a multi value field if you have the bug with the fields showing up in the attributes tab, there is no UI to create user profile fields that are multi value, right?

Exactly. On a broader scope it would be cool if for a user attribute you could enable `Multivalued` which would only then allow multi value input.

**Comment by antikalk** [comment]
@edewit Thanks for your quick response. Maybe one additional thought: Is the attributes tab even needed with the new ui when having user profile enabled?

Adding support for multi value user profile attributes might be a bigger topic, as to my knowledge this currently can not be configured. Therefor 
1. Backend functionality would have to be added (store if the attribute is multivalued), 
2. The user profile configuration screen would have to be adapted (checkbox for multivalued) and 
3. the user details screen would need to be adapted (enabling input of multiple values if the attribute is multivalued).

**Comment by edewit** [comment]
Yes I agree with the list in my PR, I only fixed the initial problem of the user profile fields appearing in the attributes tab. I'm not 100% sure, but I think attributes can still be useful even if user profile is enabled. To add the multi-value fields to the user profile is more an enhancement then a bug

**Comment by antikalk** [comment]
>  I'm not 100% sure, but I think attributes can still be useful even if user profile is enabled.

I am not sure either, but as far as I remember one could only add attributes to a user that are defined in the user profile once this feature is enabled for the realm. Either way this would only be a cosmetic thing.

> To add the multi-value fields to the user profile is more an enhancement then a bug

Makes sense, I will open a different issue for that.

**Comment by edewit** [comment]
Thanks @antikalk
